File: app/src/main/java/paul/barthuel/testrxjava/view/CountryModelDiffCallBack.java
package paul.barthuel.testrxjava.view;

import androidx.annotation.NonNull;

import paul.barthuel.testrxjava.model.CountryModel;

class CountryModelDiffCallBack extends androidx.recyclerview.widget.DiffUtil.ItemCallback<CountryModel> {

    @Override
    public boolean areItemsTheSame(@NonNull CountryModel oldItem, @NonNull CountryModel newItem) {
        return false;
    }

    @Override
    public boolean areContentsTheSame(@NonNull CountryModel oldItem, @NonNull CountryModel newItem) {
        return false;
    }
}

File: app/src/main/java/paul/barthuel/testrxjava/view/CountryModelDiffCallBack.kt
package paul.barthuel.testrxjava.view

import androidx.recyclerview.widget.DiffUtil
import paul.barthuel.testrxjava.retrofit.CountryModel

internal class CountryModelDiffCallBack : DiffUtil.ItemCallback<CountryModel>() {
    override fun areItemsTheSame(oldItem: CountryModel, newItem: CountryModel): Boolean {
        return false
    }

    override fun areContentsTheSame(oldItem: CountryModel, newItem: CountryModel): Boolean {
        return false
    }
}